src/main/java/news/caughtup/model/Article.java

```diff
@@ -19,18 +23,29 @@ public String getTitle() {
     public void setTitle(String title) {
         this.title = title;
     }
-    public String getDate() {
+    
+    public int getResourceId() {
+        return resourceId;
+    }
+
+    public void setResourceId(int resourceId) {
+        this.resourceId = resourceId;
+    }
+
+    public Date getDate() {
         return date;
     }
-    public void setDate(String date) {
+    public void setDate(Date date) {
```

Comment:
You should use `java.sql.timestamp` which is what jdbc returns for `datetime`. It works well with Gson.